lib/setup.py

```diff
@@ -38,9 +51,9 @@
                 on Sedna project",
     packages=find_packages(exclude=["tests", "*.tests",
                                     "*.tests.*", "tests.*"]),
-    author="",
-    author_email="",
-    maintainer="",
+    author=maintainer,
+    author_email="pujie2@huawei.com",
```

Comment:
Should we show the personal information here?
If we are doing advertising here, it is better to mention what is the email for.

Reply:
According to [setuptools requirements](https://docs.python.org/3.10/distutils/setupscript.html#additional-meta-data), `author_email` will be uesd to follow up on issues users encounter with the package.

Reply:
Also, as `sedna` doesn't have an official email address, providing an invalid email address just to please the tooling feels a bit weird, though.